.github/workflows/embedded_ui_refresh.yaml

```diff
@@ -0,0 +1,71 @@
+on:
+  repository_dispatch:
+    types: [embedded_ui_refresh]
+  workflow_dispatch:
+    inputs:
+      asset_name:
+        description: Release asset name
+        required: true
+        default: embedded-ui
+      repository:
+        description: Source repository
+        required: true
+        default: ydb-platform/ydb-embedded-ui
+      tag_name:
+        description: Release tag
+        required: true
+
+name: Embedded UI Refresh
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    env:
+      ASSET_NAME: ${{ github.event.inputs && github.event.inputs.asset_name || github.event.client_payload.asset_name }}
+      REPOSITORY: ${{ github.event.inputs && github.event.inputs.repository || github.event.client_payload.repository }}
+      TAG_NAME: ${{ github.event.inputs && github.event.inputs.tag_name || github.event.client_payload.tag_name }}
+      TEMP_ASSET_DIR: /tmp/asset
+    steps:
+      - uses: actions/checkout@v4
+      - name: Asset Download
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What is the difference between `GITHUB_TOKEN` and `YDB_PLATFORM_BOT_TOKEN_REPO` (used in ydb-embedded-ui workflows)

Reply:
`secrets.GITHUB_TOKEN` is a default token that is provided by GitHub for any workflow. I found that it used also in [`docs_preview`](https://github.com/ydb-platform/ydb/blob/main/.github/workflows/docs_preview.yaml) workflow.

No idea why `secrets.GITHUB_TOKEN` didn't use in `ydb-embedded-ui` for opening PRs.